civicrm.module

```diff
@@ -86,11 +86,23 @@ function civicrm_toolbar() {
         '#options' => [
           'attributes' => [
             'title' => t('CiviCRM'),
-            'class' => ['toolbar-item'],
           ],
         ],
       ],
     ];
+    // Add tray if we are on a CiviCRM screen
+    $path = ltrim(\Drupal::service('path.current')->getPath(), '/');
+    if (strpos($path, 'civicrm') === 0) {
+      $items['civicrm']['tray'] = [
+        '#heading' => t('Test tray'),
```

Comment:
Should this be removed from production code?

Reply:
Removed